controllers/books.js

```diff
@@ -1,56 +1,77 @@
 const Book = require("../models/book");
 
-const NUMBER_OF_BOOKS_TO_SHOW = 3;
+const NUMBER_OF_BOOKS_TO_SHOW = 6;
 
 const BooksController = {
   Index: (req, res) => {
     const { query } = req;
-    console.log({ query });
 
     Book.find((err, books) => {
       if (err) {
         throw err;
       }
-      console.log({ books: books.slice(0, NUMBER_OF_BOOKS_TO_SHOW) });
+      const filteredBooks = books
+        .filter((filteredByDate) => filteredByDate.discussionDate != "TBC")
+        // This needs to sort by date, but currently passing in months as a string.
+        .sort((a, b) => { return b.discussionDate - a.discussionDate});
+
+      let nextButton = false;
+      if (filteredBooks.length > NUMBER_OF_BOOKS_TO_SHOW) {
+        nextButton = true;
+      }
+
       res.render("books/index", {
-        books: books.slice(0, NUMBER_OF_BOOKS_TO_SHOW),
+        books: filteredBooks.slice(0, NUMBER_OF_BOOKS_TO_SHOW),
+        nextButton: nextButton,
+        suggestionsButton: true,
         bookContent:
-          books.filter(({ _id }) => _id === query?.selectedBook)?.[0] ||
+          books.filter(({ _id }) => _id == query?.selectedBook)?.[0] ||
           books[0] ||
           {},
       });
     });
   },
 
+  // This "more"books page is almost a repetition of the page above but not sure how else to do it with 6 results per page. 
+  More: (req, res) => {
+    const { query } = req;
+
+    Book.find((err, books) => {
+      if (err) {
+        throw err;
+      }
+      const filteredBooks = books
+        .filter((filteredByDate) => filteredByDate.discussionDate != "TBC")
+        .sort((a, b) => { return b.discussionDate - a.discussionDate})
+        .slice(6, 12);
```

Comment:
this might be a way of abstractedout some of the logic your re-using
```suggestion
      const getFilteredBooks = ({unfilteredBooks, startIndex, pageSize}) => unfilteredBooks
        .filter((filteredByDate) => filteredByDate.discussionDate != "TBC")
        .sort((a, b) => { return b.discussionDate - a.discussionDate})
        .slice(startIndex, startIndex + pageSize);
      const filteredBooks = getFilteredBooks({unfilteredBooks: books, startIndex: 6, pageSize: 6})
```

Reply:
Noted, that is easier to understand. Thanks.